lesson02/elb_asg/main.tf

```diff
@@ -0,0 +1,113 @@
+// Configure AWS Cloud provider
+provider "aws" {
+  region = "${var.aws-region}"
+}
+
+# terraform {
```

Comment:
If this code is not needed, could we delete it?

Reply:
I comment it out since i was not sure what bucket to use, I have updated it to use
    bucket = "wizeline-academy-terraform"

Reply:
That one is the right bucket :)